libnmstate/metadata.py

```diff
@@ -92,12 +92,23 @@ def _get_bond_slaves_from_state(iface_state, default=()):
 
 def _set_ovs_bridge_ports_metadata(master_state, slave_state):
     _set_common_slaves_metadata(master_state, slave_state)
-
-    ports = master_state.get('bridge', {}).get('port', [])
-    port = next(
-        six.moves.filter(lambda n: n['name'] == slave_state['name'], ports), {}
+    slave_state[BRPORT_OPTIONS] = _find_ovs_bridge_port(
+        master_state, slave_state['name']
     )
-    slave_state[BRPORT_OPTIONS] = port
+
+
+def _find_ovs_bridge_port(bridge_state, port_name):
+    for port_state in bridge_state.get('bridge', {}).get('port', []):
```

Comment:
Use schema constants for "bridge" and "port"

Reply:
Petr hates constants, but will do.